src/app/components/article/MediaArticlesCard.js

```diff
@@ -61,6 +60,8 @@ const MediaArticlesCard = ({ article, team, onAdd }) => {
           { isHovered && <AddIcon /> }
           { article.nodeType === 'Explainer' && !isHovered && <BookIcon /> }
           { article.nodeType === 'FactCheck' && !isHovered && <FactCheckIcon /> }
+          { article.nodeType === 'FactCheck' && <FormattedMessage id="articlesSidebarCard.factCheck" tagName="small" defaultMessage="Claim & Fact-Check" description="Type description of a fact-check article card." /> }
+          { article.nodeType === 'Explainer' && <FormattedMessage id="articlesSidebarCard.explainer" tagName="small" defaultMessage="Explainer" description="Type description of an explainer article card." /> }
```

Comment:
The namespace ID for these `<FormattedMessage />`'s should be `mediaArticlesCard.`